images/custom-error-pages/rootfs/www/403-refresh-fresha.html

```diff
@@ -0,0 +1,20 @@
+{
+"data":{
+"attributes":{
+"application":"fresha",
+"deprecated":false,
+"locked":true,
+"recheck-duration":null,
+"maintenance-mode":{
+"enabled":true,
+"message":"We are offline until %eta%",
+"eta":"2019-09-01T8:35:00.000Z"
```

Comment:
Can we make these things:
* enabled
* message
* eta
configurable via environment variables to be able to quickly change this?